server.js

```diff
@@ -0,0 +1,59 @@
+var express = require("express"); // expressJS is required
+var app = express();
+var bodyparser = require("body-parser"); //body-parser is required
+var piglatinify = require("./lib/piglatinify.js");
+var randomgenerator = require("./lib/randomGenerator.js");
+var randomtrivia = require("./lib/mlbTrivia.js");
+var ubbidubbify = require("./lib/ubbidubbi.js");
+var port = process.env.PORT || 3000;
+
+var quotes = [
```

Comment:
Really clean, concise server file. Just this one variable and functionality that needs to be moved into a different file - but I know you know that. Good work on getting all the other extraneous functionality packaged up into modules.